dist/prometheus.js

```diff
@@ -0,0 +1,97 @@
+const express = require('express');
+const pm2 = require('pm2');
+
+const DEBUG = /^(on|1|true|yes)$/i.test(process.env.DEBUG);
+
+const client = require('prom-client');
+const requestCounts = new client.Gauge({
+  name: 'http_request_counts',
+  help: 'counter for requests labeled with: method, status_code, theme, base_href, path',
+  labelNames: ['method', 'status_code', 'theme', 'base_href', 'path'],
+});
+const requestDuration = new client.Histogram({
+  name: 'http_request_duration_seconds',
+  help: 'duration histogram of http responses labeled with: status_code, theme',
+  buckets: [0.1, 0.3, 0.5, 0.7, 0.9, 1.1, 1.3, 1.5, 2, 3, 4, 5, 6, 7, 8, 9, 10, 20, 30],
+  labelNames: ['status_code', 'theme', 'base_href', 'path'],
+});
+const up = new client.Gauge({
+  name: 'up',
+  help: '1 = up, 0 = not up',
+});
+up.set({}, 1);
+const pm2Processes = new client.Gauge({
+  name: 'pm2_processes',
+  help: 'counter for pm2 processes',
+  labelNames: ['name'],
+});
+const pm2ProcessRestarts = new client.Gauge({
+  name: 'pm2_process_restarts',
+  help: 'counter for pm2 process restarts',
+  labelNames: ['name'],
+});
+const pm2Memory = new client.Gauge({
+  name: 'pm2_memory',
+  help: 'counter for pm2 memory',
+  labelNames: ['name'],
+});
+
+const app = express();
+
+app.post('/report', (req, res) => {
```

Comment:
Who is posting json to this resource?

Reply:
The SSR process:

https://github.com/intershop/intershop-pwa/blob/0045dddc15657710407daac08e195b19a686575b/server.ts#L340